src/add.ts

```diff
@@ -15,15 +16,12 @@ type Paths = {
   sloped: Path[];
 };
 
+type Queues = {
+  queue: Path[];
+}
+
 export function Add(settings: Settings): Paths {
   const paths: Paths = { unsloped: [], sloped: [] };
-  const benchIndexes = settings.benches.map((b: LoadedObject) => b.index);
```

Comment:
benchIndexes and binIndexes were only used within that conflictsWithExistingAddition function, which I made into its own function outside of Add since I was reusing it for addQueueTVs. I changed it a bit too, as this was causing the issue with preserving existing path additions. In hindsight I probably should have made a separate branch for each issue, apologies there.